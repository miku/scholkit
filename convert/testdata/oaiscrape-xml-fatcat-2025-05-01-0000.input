<record xmlns="http://www.openarchives.org/OAI/2.0/"><header status=""><identifier>oai:ojs.pkp.sfu.ca:article/1321</identifier><datestamp>2021-10-02T18:23:53Z</datestamp><setSpec>jecs:EXP</setSpec><setSpec>driver</setSpec></header><metadata>
<oai_dc:dc
xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/"
xmlns:dc="http://purl.org/dc/elements/1.1/"
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/
http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
<dc:title xml:lang="en-US">Pedagogical Design of the Technology of Students’ Multicultural Competence at Higher Education Institutions</dc:title>
<dc:creator>Kravets, Ruslan</dc:creator>
<dc:creator>Vykhrushch, Vira </dc:creator>
<dc:creator>Romanyshyna, Oksana </dc:creator>
<dc:creator>Koziar, Mykhailo </dc:creator>
<dc:creator>Ridkodubska, Hanna </dc:creator>
<dc:creator>Marionda, Ivan </dc:creator>
<dc:creator>Syvokhop, Eduard </dc:creator>
<dc:subject xml:lang="en-US">Professional training </dc:subject>
<dc:subject xml:lang="en-US">multicultural education</dc:subject>
<dc:subject xml:lang="en-US">multicultural competence</dc:subject>
<dc:subject xml:lang="en-US">pedagogical technology</dc:subject>
<dc:description xml:lang="en-US">Aim. The aim of the research is to justify the theoretical and methodological bases of multicultural education and verify experimentally the efficiency of the pedagogical technology of forming students’ multicultural competence at higher education institutions.
Methods. The study reviews the changes in the levels of students’ multicultural competence. The methodological bases of the study are theories of language multicultural education, the unity of language and culture, multi-perspective learning, “cultural differences,” social learning, neo-Freudian theory, self-determination theory, developmental learning theory.
Results. The realisation of the ideas of multiculturalism has a positive effect on the harmonious development, cultivates a tolerant attitude to other ethnic groups and peoples, promotes good neighbourly relations, effective conflict resolution, intercultural dialogue and mutual understanding. Acquiring these characteristics, students entirely understand national and universal values, and are able to interact constructively in the modern multicultural world at a sustainable level.
Conclusions. The implementation of the developed pedagogical conditions and the author’s pedagogical technology of forming students’ multicultural competence at higher education institutions contributed to a significant increase in the levels of their multicultural competence. This was confirmed by statistical changes in the level of students’ multicultural competence of the experimental group compared to the control group: the number of students with stable (by 34,3%) and situational (by 12,7%) levels significantly increased, the number of students with fragmentary level decreased (by 47%). Thus, the obtained results prove that the developed pedagogical technology is effective for forming students’ multicultural competence.</dc:description>
<dc:publisher xml:lang="en-US">Foundation Pro Scientia Publica/ University of Wrocław</dc:publisher>
<dc:date>2021-09-25</dc:date>
<dc:type>info:eu-repo/semantics/article</dc:type>
<dc:type>info:eu-repo/semantics/publishedVersion</dc:type>
<dc:type xml:lang="en-US">text</dc:type>
<dc:format>application/pdf</dc:format>
<dc:identifier>https://e-methodology.eu/index.php/jecs/article/view/1321</dc:identifier>
<dc:identifier>10.15503/jecs2021.2.264.293</dc:identifier>
<dc:source xml:lang="en-US">Journal of Education Culture and Society; Vol. 12 No. 2 (2021): Journal of Education Culture and Society; 264-293</dc:source>
<dc:source>2081-1640</dc:source>
<dc:source>2081-1640</dc:source>
<dc:language>eng</dc:language>
<dc:relation>https://e-methodology.eu/index.php/jecs/article/view/1321/1167</dc:relation>
<dc:rights xml:lang="en-US">Copyright (c) 2021 Ruslan Kravets</dc:rights>
<dc:rights xml:lang="en-US">http://creativecommons.org/licenses/by/4.0</dc:rights>
</oai_dc:dc>
</metadata><about></about></record>

