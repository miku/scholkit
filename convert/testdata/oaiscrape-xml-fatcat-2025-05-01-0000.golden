{
    "source": "oaiscrape",
    "contribs": [
        {
            "extra": {},
            "raw_name": "Kravets, Ruslan"
        },
        {
            "extra": {},
            "raw_name": "Vykhrushch, Vira"
        },
        {
            "extra": {},
            "raw_name": "Romanyshyna, Oksana"
        },
        {
            "extra": {},
            "raw_name": "Koziar, Mykhailo"
        },
        {
            "extra": {},
            "raw_name": "Ridkodubska, Hanna"
        },
        {
            "extra": {},
            "raw_name": "Marionda, Ivan"
        },
        {
            "extra": {},
            "raw_name": "Syvokhop, Eduard"
        }
    ],
    "ext_ids": {
        "doi": "10.15503/jecs2021.2.264.293",
        "oai": "oai:ojs.pkp.sfu.ca:article/1321"
    },
    "extra": {
        "arxiv": {},
        "dblp": {},
        "openalex": {
            "open_access": {}
        },
        "crossref": {},
        "oai": {
            "set_spec": [
                "jecs:EXP",
                "driver"
            ],
            "url": [
                "https://e-methodology.eu/index.php/jecs/article/view/1321"
            ]
        }
    },
    "id": "oaiscrape-8e3a106e38de6c97bc3fbf6bd85f99979f27498b",
    "language": "eng",
    "release_date": "2021-09-25",
    "title": "Pedagogical Design of the Technology of Studentsâ€™ Multicultural Competence at Higher Education Institutions"
}